jekyll/_cci2/android-images-support-policy.adoc

```diff
@@ -0,0 +1,94 @@
+
+We install the most up-to-date versions of each tool/package in newly built images.
+
+We aim to package 6 levels of the Android API within each image release.
+
+As this image is not specifically designed for the Android Gradle Plugin (AGP), some updates may come later than desired. However, we aim to follow compatibility matrices for https://docs.gradle.org/current/userguide/compatibility.html and https://developer.android.com/build/releases/gradle-plugin#updating-gradle.
+
+Releases may be skipped if there are no material updates to core Android functionality, such as sdkmanager, ndk or platform versions.
+
+[#tagging]
+== Tagging
+
+Tags we support for this image (tag is what is specified in config when requesting an image):
+
+For the latest major version of Docker:
+
+- `default`: This image is the current stable version and of image will receive updates approximately every 3 months.
+
+- `edge`: These tags are reserved for previews of new releases, which will initially point to this tag. The `edge` tags may include incremental updates to the `current` image release, which may change without notice, and is not recommended to be used for production CI workloads. currentwill be updated with these changes after period of stability (generally an average of a week).
+
+- Date based tagging: Android images are generally released once every 3 months. The date-based tagging conventions take the format of <year>.<month>.<patch>, such as 2023.10.1. We recommend using the default version and not pinning to a date version.
+
+[#critical-cve-patches]
+== Critical CVE patches
+
+When critical CVEs are disclosed that affect the versions of the operating system or software stack in our Docker Convenience Images, we will investigate the impact that this has on our images being used within the CircleCI execution environment. If customers are impacted by these CVEs we will push a patch fix to the released image(s), this image will supersede the original image
+
+[#bug-reports-issues-and-prs]
+== Bug reports, issues, and PRs
+
+You can file a [ticket]https://support.circleci.com/hc/en-us/requests/new with CircleCI Support for any issues or bugs found with the remote Docker images. Our support team will be able to escalate issues internally to the correct engineering team and provide updates on the issue.
+
+[#image-lifespan-eol]
+== Image lifespan / EOL
+
+We will keep a total of 7 images at maximum, with a deprecation cycle happening once per year. The table below shows what images will be kept in each cycle. Generally we will aim to start EOL process within 3 months of new version release.
+
+Current Deprecation (Jan 2024):
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+| Image
+| Lowest API Level Support
+
+| Android
+| API Level 27 (Android 8); Retain Q4 release of previous two years
+
+|===
+
+Example: When Android 15 releases:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
```

Comment:
This does not match what is in policy we got sign off for from support and others?

Reply:
will do